unit-tests/src/test/java/org/eclipse/collections/impl/collector/Collectors2Test1Test.java

```diff
@@ -18,47 +18,18 @@
 
 import org.eclipse.collections.api.collection.primitive.MutableIntCollection;
 import org.eclipse.collections.api.list.MutableList;
-import org.eclipse.collections.api.list.primitive.BooleanList;
-import org.eclipse.collections.api.list.primitive.ByteList;
-import org.eclipse.collections.api.list.primitive.CharList;
-import org.eclipse.collections.api.list.primitive.DoubleList;
-import org.eclipse.collections.api.list.primitive.FloatList;
-import org.eclipse.collections.api.list.primitive.IntList;
-import org.eclipse.collections.api.list.primitive.LongList;
-import org.eclipse.collections.api.list.primitive.ShortList;
 import org.eclipse.collections.api.map.primitive.MutableObjectLongMap;
 import org.eclipse.collections.api.multimap.bag.MutableBagMultimap;
 import org.eclipse.collections.api.partition.PartitionMutableCollection;
-import org.eclipse.collections.api.partition.bag.PartitionMutableBag;
-import org.eclipse.collections.api.partition.list.PartitionMutableList;
-import org.eclipse.collections.api.partition.set.PartitionMutableSet;
-import org.eclipse.collections.impl.block.factory.Functions;
-import org.eclipse.collections.impl.block.factory.IntegerPredicates;
-import org.eclipse.collections.impl.block.factory.Predicates2;
-import org.eclipse.collections.impl.factory.Bags;
 import org.eclipse.collections.impl.factory.BiMaps;
-import org.eclipse.collections.impl.factory.Lists;
-import org.eclipse.collections.impl.factory.Sets;
 import org.eclipse.collections.impl.factory.Stacks;
-import org.eclipse.collections.impl.factory.primitive.BooleanLists;
-import org.eclipse.collections.impl.factory.primitive.ByteLists;
-import org.eclipse.collections.impl.factory.primitive.CharLists;
-import org.eclipse.collections.impl.factory.primitive.DoubleLists;
-import org.eclipse.collections.impl.factory.primitive.FloatLists;
 import org.eclipse.collections.impl.factory.primitive.IntBags;
-import org.eclipse.collections.impl.factory.primitive.IntLists;
-import org.eclipse.collections.impl.factory.primitive.LongLists;
-import org.eclipse.collections.impl.factory.primitive.ShortLists;
 import org.eclipse.collections.impl.list.Interval;
 import org.eclipse.collections.impl.partition.bag.PartitionHashBag;
-import org.eclipse.collections.impl.partition.list.PartitionFastList;
-import org.eclipse.collections.impl.partition.set.PartitionUnifiedSet;
-import org.eclipse.collections.impl.test.Verify;
-import org.eclipse.collections.impl.tuple.primitive.PrimitiveTuples;
 import org.junit.Assert;
 import org.junit.Test;
 
-public final class Collectors2Test
+public final class Collectors2Test1Test
```

Comment:
The file name needs to end with 'Test' because of the sure-fire plugin config:

 <include>**/*Test.java</include>